### ISSUE TITLE
Problem with sending a new fleet to attack another player

### ISSUE THREAD

**Issue by Nusuminat** [koraykarakus/steemnova-1.8-x#18]
Hello . I added a new ship to the game. and now a problem has arisen because you can transfer this ship to another planet or moon without any problem, or transfer it to another player without any problem, but when you want to attack another player, the fleet reaches the target and hangs. even unlocking does not give any result. what could be the problem? the ship is visible, you can build it or unlock it through research without any problems.

**Comment by koraykarakus** [comment]
includes/libs/opbe/errors 

check the above path for opbe errors if any. If not any error there check admin panel and tickets page. Without seeing type of error it is hard to tell.

**Comment by Nusuminat** [comment]
Thank you for the tip. I found an error. the problem was that there were missing entries in one language directory Thursday 7th of November 2024 11:10:06 PM Unknown error type: [2] Undefined array key 280 Error on line 21 in file includes/classes/missions/functions/LangImplementation. php, 
PHP 8.2.30 (Linux) $_POST = $_GET =

30% - 30% - 10%
array ( )array ( 'page' => 'overview', ) I looked for the reason in error.log but there was nothing there. now everything works, the fleet doesn't hang up.
